js/stat.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var CLOUD_GAP = 10;
+var FONT_GAP = 20;
+var TEXT_HEIGHT = 35;
+var BAR_WIDTH = 40;
+var BAR_MAX_HEIGHT = 150;
+var BAR_GAP = 50;
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function (arr) {
+  var maxElement = arr[0];
+
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > maxElement) {
+      maxElement = arr[i];
+    }
+  }
+
+  return maxElement;
+};
```

Comment:
Вместо цикла for лучше использовать более подходящий для этого способ - метод reduce
```suggestion
var getMaxElement = function (items) {
  return items.reduce(function(acc, cur) {
    return cur > acc ? cur : acc;
  }, -Infinity)
};
```

Либо можно использовать метод Math.max, но он принимает не массив а последовательность значений. Эту проблему можно обойти с помощью метода apply
```suggestion
var getMaxElement = function (items) {
  return Math.max.apply(null, items)
};
```

Reply:
В случае с reduce (выбираю его, тк он более понятный) - что означает и как работает строка -Infinity?

Reply:
А если мы передадим пустой массив, то функция вернет -Infinity (минус бесконечность). Так например работает и встроенная функция Math.max() без аргументов